<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QT_UI::QMenuBarEx" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>Project(&amp;P)</string>
    </property>
    <widget class="QMenu" name="m_menuRecents">
     <property name="title">
      <string>Recents</string>
     </property>
     <addaction name="m_actionEmptyProject"/>
    </widget>
    <widget class="QMenu" name="m_menuSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <addaction name="m_actionEmptyProject"/>
    </widget>
    <addaction name="m_actionNewProject"/>
    <addaction name="m_actionOpenProject"/>
    <addaction name="m_menuRecents"/>
    <addaction name="separator"/>
    <addaction name="m_menuSettings"/>
    <addaction name="separator"/>
    <addaction name="m_actionValiateData"/>
    <addaction name="separator"/>
    <addaction name="m_actionSaveProject"/>
    <addaction name="m_actionSaveAsProject"/>
    <addaction name="separator"/>
    <addaction name="m_actionExitEditor"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="m_actionDocumentation"/>
    <addaction name="m_actionApi"/>
    <addaction name="separator"/>
    <addaction name="m_actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game(&amp;G)</string>
    </property>
    <addaction name="m_actionPlayGame"/>
    <addaction name="m_actionStopGame"/>
   </widget>
   <widget class="QMenu" name="menuScene_S">
    <property name="title">
     <string>Scene(&amp;S)</string>
    </property>
    <addaction name="m_actionNewScene"/>
    <addaction name="separator"/>
    <addaction name="m_actionSaveScene"/>
    <addaction name="m_actionSaveAsScene"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuScene_S"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="m_toolBar">
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="m_actionAbout">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/icon.png</normaloff>:/icon/Icon/icon.png</iconset>
   </property>
   <property name="text">
    <string>About Echo</string>
   </property>
  </action>
  <action name="m_actionSaveProject">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/project/save.png</normaloff>:/icon/Icon/project/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_actionPlayGame">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/game/play.png</normaloff>:/icon/Icon/game/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="m_actionStopGame">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/game/stop.png</normaloff>:/icon/Icon/game/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="m_actionExitEditor">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="m_actionNewScene">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="m_actionSaveAsScene">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="m_actionSaveScene">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="m_actionSaveAsProject">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="m_actionNewProject">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_actionOpenProject">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_actionEmptyProject">
   <property name="text">
    <string>Empty Project</string>
   </property>
  </action>
  <action name="m_actionApi">
   <property name="text">
    <string>API Reference</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="m_actionValiateData">
   <property name="text">
    <string>Valiate Data ...</string>
   </property>
  </action>
  <action name="m_actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QT_UI::QMenuBarEx</class>
   <extends>QMenuBar</extends>
   <header>qmenubarex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resource/App.qrc"/>
 </resources>
 <connections/>
</ui>
